docs/IRISMapCube.rst

```diff
@@ -0,0 +1,94 @@
+===========
+IRISMapCube
+===========
+
+This class represents Slit Jaw Images from IRIS described by a single World Coordinate
+System (WCS).
+
+Initialization
+--------------
+
+To initialize an IRISMapCube, we will need to open a level 2 fits file. You can find one
+of these files in the IRIS_ website by selecting a file either by clicking on the map or by
+setting some information in the left panel. Now that we get a level 2 fits file, we can
+load it into the reading method for fits files. This method will read all the fits file and
+will take every information that it needs to create an IRISMapCube object.
+
+Let assume that we will call our fits file ``my_fits_file`` and IRISMapCube object as
+``my_cube``: ::
+
+    >>> from irispy import read_iris_sji_level2_fits
+    >>> my_cube = read_iris_sji_level2_fits(my_fits_file)
+
+If you don't have a lot of RAM memory of if you are loading a huge file, we recommend to
+use the memmap kwarg. By using it, you will only load what you need to run but some
+methods that requires all the file will not be accessible. You can use the memmap
+kwarg by doing: ::
+
+    >>> from irispy import read_iris_sji_level2_fits
+    >>> my_cube = read_iris_sji_level2_fits(my_fits_file, memmap=True)
+
+So now, ``my_cube`` is an IRISMapCube object and we can access to a lot of information like:
+
+- ``my_cube.data`` : In this attribute, we can find the data array.
+- ``my_cube.wcs`` : This attribute contains the WCS.
+- ``my_cube.mask`` : This attribute can be used to mask some data (eg. the dust particle).
```

Comment:
There should be a paragraph discussing the uncertainty attribute and how the FITS reader function calculates it.

Reply:
Added